src/preludes/ria.ae

```diff
@@ -56,6 +56,34 @@ theory Real_of_Int extends RIA =
      {i <= real_of_int(x)}.
      k <= x
 
+  (* floor(x) ≤ i iff x < i + 1 *)
```

Comment:
I guess you meant `floor(x) <= i implies x < i+1` instead of `floor(x) <= i iff x < i+1`. 
Besides, I don't understand why you need to use the semantic trigger `not_theory_constant` here?

Reply:
`floor(x) <= i iff x < i + 1` is correct, unless I am mistaken? Even though we only implement the implication.

`not_theory_constant` is here to avoid adding useless axioms, e.g. `3 < 4` if we know that `floor(3.5) <= 3`